src/SourceBuild/content/eng/tools/join-verticals.proj

```diff
@@ -0,0 +1,46 @@
+<Project Sdk="Microsoft.Build.NoTargets">
```

Comment:
Style: Indentation is wrong in this file. We use two spaces everywhere else in msbuild files.